### PR TITLE
Add mmdet3d doc

### PR DESCRIPTION
**Pull request by tpoisonooo**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation and Modification

* Add mmdet3d doc
* Update mmdet3d docstring
* mmrotate typo

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/04-supported-codebases/mmdet3d.md [hunk 1]
```diff
@@ -1,33 +1,99 @@
...
 
 ______________________________________________________________________
 
 [MMDetection3d](https://github.com/open-mmlab/mmdetection3d) aka `mmdet3d` is an open source object detection toolbox based on PyTorch, towards the next-generation platform for general 3D detection. It is a part of the [OpenMMLab](https://openmmlab.com/) project.
 
-## Installation
+## Install mmdet3d
 
-### Install mmdet3d
+These branches are required for mmdet3d deployment
```
Comment (RunningLeon) [inline docs/en/04-supported-codebases/mmdet3d.md:14]: better to use mim install mmdet3d
eg
```
python -m pip install openmim
python -m mim install 'mmdet3d>=v1.1.0rc1'
```

File: docs/en/04-supported-codebases/mmdet3d.md [hunk 2]
```diff
@@ -1,33 +1,99 @@
...
+..
+11/11 13:56:19 - mmengine - INFO - mmdet3d:     1.1.0rc1
+```
 
 ## Convert model
 
-## Model specification
+For example, use `tools/deploy.py` to convert centerpoint to onnxruntime format
+
+```bash
```
Comment (RunningLeon) [inline docs/en/04-supported-codebases/mmdet3d.md:63]: use mim to dowload config and ckpt

File: docs/en/04-supported-codebases/mmdet3d.md (hunk 1 above)
Comment (tpoisonooo) [reply docs/en/04-supported-codebases/mmdet3d.md:14]: done.

File: docs/en/04-supported-codebases/mmdet3d.md (hunk 2 above)
Comment (tpoisonooo) [reply docs/en/04-supported-codebases/mmdet3d.md:61]: mim not support mmdet3d=1.1.0rc1 and mmdet3d dev3d-1.x has changed config name.

File: docs/en/04-supported-codebases/mmdet3d.md [hunk 3]
```diff
@@ -1,33 +1,87 @@
...
 
 ## Model inference
 
-### Backend model inference
+At present, the voxelize preprocessing and postprocessing of mmdet3d are not converted into onnx operations; the C++ SDK has not yet implemented the voxelize calculation.
+
+The caller needs to refer to the corresponding [python implementation](../../../mmdeploy/codebase/mmdet3d/deploy/voxel_detection_model.py) to complete.
 
-### SDK model inference
+mmdeploy currently not merge preprocess and postprocess of mmdet3d into onnx, and C++ SDK has not yet implemented voxelize operation.
```
Comment (lvhan028) [inline docs/en/04-supported-codebases/mmdet3d.md:75]: Duplicate description which has been presented at the beginning of this chapter

**Review by RunningLeon — APPROVED** [review]
LGTM after resolve left comments.

File: docs/en/04-supported-codebases/mmdet3d.md (hunk 3 above)
Comment (tpoisonooo) [reply docs/en/04-supported-codebases/mmdet3d.md]: eon.

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**